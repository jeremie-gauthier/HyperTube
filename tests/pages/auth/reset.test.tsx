import React from "react";
import Reset from "@/pages/auth/reset";
import { render } from "@testing-library/react";

test("<Reset /> should render correctly", () => {
  const tree = render(<Reset />);
  expect(tree).toMatchSnapshot();
});
