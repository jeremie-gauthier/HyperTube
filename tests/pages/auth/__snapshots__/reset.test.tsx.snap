// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reset /> should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="container"
      >
        <div
          class="scrollbar-container scrollContainer ps"
        >
          <h1>
            pages.auth.reset.forgotten_password
          </h1>
          <form
            class="authForm mt-4"
          >
            <div
              class="container"
            >
              <input
                autocomplete="off"
                class=" border-grey-light"
                id="email"
                name="email"
                placeholder="pages.auth.reset.associated_email"
                value=""
              />
            </div>
            <button
              type="submit"
            >
              common.buttons.submit
            </button>
          </form>
          <div
            class="authLinksGroup"
          >
            <div
              class="switchAuthLink"
            >
              <p>
                pages.auth.login.first_visit
              </p>
              <a
                href="/auth/register"
              >
                pages.auth.login.register
              </a>
            </div>
            <div
              class="switchAuthLink"
            >
              <p>
                pages.auth.register.already_registered
              </p>
              <a
                href="/auth/login"
              >
                pages.auth.register.login
              </a>
            </div>
          </div>
          <div
            class="ps__rail-x"
            style="left: 0px; top: 0px;"
          >
            <div
              class="ps__thumb-x"
              style="left: 0px; width: 0px;"
              tabindex="0"
            />
          </div>
          <div
            class="ps__rail-y"
            style="top: 0px; left: 0px;"
          >
            <div
              class="ps__thumb-y"
              style="top: 0px; height: 0px;"
              tabindex="0"
            />
          </div>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="container"
    >
      <div
        class="scrollbar-container scrollContainer ps"
      >
        <h1>
          pages.auth.reset.forgotten_password
        </h1>
        <form
          class="authForm mt-4"
        >
          <div
            class="container"
          >
            <input
              autocomplete="off"
              class=" border-grey-light"
              id="email"
              name="email"
              placeholder="pages.auth.reset.associated_email"
              value=""
            />
          </div>
          <button
            type="submit"
          >
            common.buttons.submit
          </button>
        </form>
        <div
          class="authLinksGroup"
        >
          <div
            class="switchAuthLink"
          >
            <p>
              pages.auth.login.first_visit
            </p>
            <a
              href="/auth/register"
            >
              pages.auth.login.register
            </a>
          </div>
          <div
            class="switchAuthLink"
          >
            <p>
              pages.auth.register.already_registered
            </p>
            <a
              href="/auth/login"
            >
              pages.auth.register.login
            </a>
          </div>
        </div>
        <div
          class="ps__rail-x"
          style="left: 0px; top: 0px;"
        >
          <div
            class="ps__thumb-x"
            style="left: 0px; width: 0px;"
            tabindex="0"
          />
        </div>
        <div
          class="ps__rail-y"
          style="top: 0px; left: 0px;"
        >
          <div
            class="ps__thumb-y"
            style="top: 0px; height: 0px;"
            tabindex="0"
          />
        </div>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
